Fix copy editing

Gyan-max opened the pull request:
Description:
While browsing the Qiskit documentation, I found some inconsistencies in formatting and terminology:

The word "Addons" appears capitalized inconsistently. It should be reviewed for uniformity.
The term "Intro" is used in some places where "Introduction" would be more appropriate.
The phrase "circuit building" should be hyphenated to maintain consistency with standard grammar rules (i.e., "circuit-building").
Suggested Fixes:
Standardize "Addons" usage across the documentation.
Replace "Intro" with "Introduction" wherever applicable.
Change "circuit building" to "circuit-building" for proper hyphenation.

frankharkins commented on docs/make.bat:
Please remove this too

frankharkins reviewed (changes requested):
@Gyan-max thanks for this! Close reviews of the content like this one are really helpful. The copy edits look good to me, but I'll get one of our writers to double-check.

Only two problems are (1) I don't think we're likely to be able to rename any pages at the moment, and (2) I think you've included some files by accident. See my review comments for more details.

Gyan-max commented:
"I have addressed all the requested changes:

Reverted the unintended filename change and restored _toc.json.
Reset noise-learning.ipynb to its original state.
Removed unnecessary files (Makefile, _build, source, make.bat).
Fixed the capitalization inconsistency in OpenQASM references. Let me know if any further adjustments are needed!"

abbycross commented on docs/guides/addons.mdx:
```diff
@@ -5,9 +5,9 @@ description: Understand the Qiskit addon tools, which help you build utility-gra
+Qiskit addons are a collection of research capabilities for enabling algorithm discovery at the utility scale. These capabilities build on Qiskit's performant foundation of tools for creating and running quantum algorithms. They are provided as modular software components that can plug into a [workflow](/guides/introduction-to-patterns) to scale or design new quantum algorithms.
 
-## Addons for mapping
+## addons for mapping
```
We use the capital A here not because of the product name, but because it is the first word in a header, and that takes precedence in this case.
```suggestion
## Addons for mapping
```

abbycross commented on docs/guides/addons.mdx:
```diff
@@ -24,7 +24,7 @@ Multi-product formulas (MPF) reduce the Trotter error of Hamiltonian dynamics th
 - Read the [documentation.](https://qiskit.github.io/qiskit-addon-mpf)
 
-## Addons for optimizing
+## addons for optimizing
```
```suggestion
## Addons for optimizing
```

abbycross commented on docs/guides/addons.mdx:
```diff
@@ -41,7 +41,7 @@ Circuit cutting reduces the depth of transpiled circuits by decomposing entangli
 - Read the [documentation.](https://qiskit.github.io/qiskit-addon-cutting)
 
-##  Addons for post-processing
+##  addons for post-processing
```
```suggestion
##  Addons for post-processing
```

abbycross commented on docs/guides/index.mdx:
```diff
@@ -100,7 +100,7 @@ pip install qiskit-ibm-transpiler
 ## Qiskit addons
 
-Qiskit addons are a collection of research capabilities for utility-scale algorithm discovery. These capabilities build upon Qiskit’s performant foundation of tools for creating and running quantum algorithms. Addons are modular software components that plug into a workflow to scale or design new quantum algorithms. To learn more about the set of available Qiskit addons and how to get started usi ...
+Qiskit addons are a collection of research capabilities for utility-scale algorithm discovery. These capabilities build upon Qiskit’s performant foundation of tools for creating and running quantum algorithms. addons are modular software components that plug into a workflow to scale or design new quantum algorithms. To learn more about the set of available Qiskit addons and how to get started usi ...
```
```suggestion
Qiskit addons are a collection of research capabilities for utility-scale algorithm discovery. These capabilities build upon Qiskit’s performant foundation of tools for creating and running quantum algorithms. Addons are modular software components that plug into a workflow to scale or design new quantum algorithms. To learn more about the set of available Qiskit addons and how to get started using them, visit the [documentation](/guides/addons).
```

abbycross replied:
Need a capital letter at the beginning of the sentence, which in this case takes precedence over the product name.

abbycross commented on docs/guides/_toc.json:
```diff
@@ -48,7 +48,7 @@
             {
               "title": "Introduction to Qiskit patterns",
-              "url": "/guides/intro-to-patterns"
+              "url": "/guides/introduction-to-patterns"
```
```suggestion
              "url": "/guides/intro-to-patterns"
```
A change to the file name will break the table of contents link and other referring links - restoring this for now

abbycross commented on scripts/js/commands/checkQiskitBotFiles.ts:
```diff
@@ -32,7 +32,7 @@ const ALLOWED_OWNERLESS_FILES = new Set([
   "docs/guides/visualize-results",
   "docs/guides/create-transpiler-plugin",
-  "docs/guides/intro-to-patterns",
+  "docs/guides/introduction-to-patterns",
```
```suggestion
  "docs/guides/intro-to-patterns",
```

abbycross commented on scripts/config/internal-links.json:
```diff
@@ -984,7 +984,7 @@
   },
   {
-    "path": "docs/guides/intro-to-patterns.mdx",
+    "path": "docs/guides/introduction-to-patterns.mdx",
```
```suggestion
    "path": "docs/guides/intro-to-patterns.mdx",
```

abbycross commented on docs/guides/serverless.mdx:
```diff
@@ -15,7 +15,7 @@ Premium users can build, deploy, and run their workloads remotely on classical c
 ## When should I use Qiskit Serverless?
 
-Qiskit Serverless helps manage classical and quantum resources across the entire [Qiskit pattern](./intro-to-patterns) workflow. This includes some of the following examples:
+Qiskit Serverless helps manage classical and quantum resources across the entire [Qiskit pattern](./introduction-to-patterns) workflow. This includes some of the following examples:
```
```suggestion
Qiskit Serverless helps manage classical and quantum resources across the entire [Qiskit pattern](./intro-to-patterns) workflow. This includes some of the following examples:
```

beckykd replied:
Thanks Abby!

abbycross commented on docs/guides/_toc.json:
```diff
@@ -185,7 +185,7 @@
                   "children": [
                     {
-                      "title": "Intro to OpenQASM",
+                      "title": "Introduction to OpenQASM",
```
If we change it in the toc we need also to change the title on the file itself. The title in the toc should match the title on the page. I will suggest that change further down

abbycross commented on docs/guides/intro-to-patterns.mdx:
```diff
@@ -1,84 +0,0 @@
----
```
Please restore the file name

Gyan-max replied:
Restore deleted file docs/guides/intro-to-patterns.mdx

abbycross commented on docs/guides/map-problem-to-circuits.mdx:
```diff
@@ -41,10 +41,10 @@ The output of this step in a Qiskit pattern is normally a collection of circuits
 * [Pulse schedules](./pulse)
 * OpenQASM
-  * [Intro to OpenQASM](./introduction-to-qasm)
+  * [introduction to OpenQASM](./introduction-to-qasm)
```
```suggestion
  * [Introduction to OpenQASM](./introduction-to-qasm)
```

abbycross commented on docs/guides/serverless-manage-resources.ipynb:
```diff
@@ -46,7 +46,7 @@
    "metadata": {},
    "source": [
-    "Qiskit Serverless allows you to manage compute and data across your [Qiskit pattern](./intro-to-patterns), including CPUs, QPUs, and other compute accelerators.\n",
+    "Qiskit Serverless allows you to manage compute and data across your [Qiskit pattern](./introduction-to-patterns), including CPUs, QPUs, and other compute accelerators.\n",
```
```suggestion
    "Qiskit Serverless allows you to manage compute and data across your [Qiskit pattern](./intro-to-patterns), including CPUs, QPUs, and other compute accelerators.\n",
```

abbycross commented on docs/guides/map-problem-to-circuits.mdx:
```diff
@@ -41,10 +41,10 @@ The output of this step in a Qiskit pattern is normally a collection of circuits
 * [Pulse schedules](./pulse)
 * OpenQASM
-  * [Intro to OpenQASM](./introduction-to-qasm)
+  * [Introduction to OpenQASM](./Introduction-to-qasm)
```
```suggestion
  * [Introduction to OpenQASM](./intro-to-qasm)
```

abbycross commented on docs/guides/intro-to-patterns.mdx:
```diff
@@ -2,83 +2,21 @@
 
 
-
 ## Execute on target hardware
```
We're missing the rest of this page - please restore?

abbycross commented on docs/guides/index.mdx:
```diff
@@ -100,7 +100,7 @@ pip install qiskit-ibm-transpiler
 ## Qiskit addons
 
-Qiskit addons are a collection of research capabilities for utility-scale algorithm discovery. These capabilities build upon Qiskit’s performant foundation of tools for creating and running quantum algorithms. Addons are modular software components that plug into a workflow to scale or design new quantum algorithms. To learn more about the set of available Qiskit addons and how to get started usi ...
+Qiskit addons are a collection of research capabilities for utility-scale algorithm discovery. These capabilities build upon Qiskit’s performant foundation of tools for creating and running quantum algorithms. Adons are modular software components that plug into a workflow to scale or design new quantum algorithms. To learn more about the set of available Qiskit addons and how to get started usin ...
```
```suggestion
Qiskit addons are a collection of research capabilities for utility-scale algorithm discovery. These capabilities build upon Qiskit’s performant foundation of tools for creating and running quantum algorithms. Addons are modular software components that plug into a workflow to scale or design new quantum algorithms. To learn more about the set of available Qiskit addons and how to get started using them, visit the [documentation](/guides/addons).
```

Gyan-max commented:
Hi @beckykd @abbycross 
I appreciate the opportunity to contribute to this issue. However, due to other commitments, I won't be able to continue working on it. If anyone else wants to take over, please feel free.

Thank you for your time and support!

abbycross reviewed (commented):
requested changes below

abbycross commented:
> Hi @beckykd @abbycross I appreciate the opportunity to contribute to this issue. However, due to other commitments, I won't be able to continue working on it. If anyone else wants to take over, please feel free.
> 
> Thank you for your time and support!

Thank you @Gyan-max!
